tests/test_cli.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pathlib
+
+import pytest
+
+
+@pytest.fixture
+def cmd():
+    return pathlib.Path(__file__).parent.parent / "newhelm" / "main.py"
+
+
+def test_main(cmd):
+    assert os.system(f"python {cmd}") == 0
+
+
+def test_list_plugins(cmd):
+    assert os.system(f"python {cmd} list") == 0
+
+
+@pytest.mark.requires_plugins
+@pytest.mark.parametrize("sut", ["DemoMultipleChoiceSUT"])
+@pytest.mark.parametrize("test", ["demo_01", "demo_02"])
```

Comment:
All three demo Tests currently reach out to our cloud bucket to get data ([example](https://github.com/mlcommons/newhelm/blob/ab5306bc80943243cd94db7952f57d527bfc99cb/demo_plugin/newhelm/tests/demo_01_simple_qa_test.py#L34)). I think we should either make them hermetic during testing, or move this test to a once-a-day schedule.

Reply:
Good point. I also added two other versions of this #125 and #126 if you prefer that route instead. Let me know and then I can move whichever one(if any at all) you like and I'll create a scheduled workflow for it. Thanks for the feedback.

Reply:
The other two solutions require us to add a new technology. It seems like pytest has pretty good support for tests that default to off, so I don't think its worth adding the bats/bash complexities.

Do you think this gets any cleaner if we switch to skipif similar to [this recommendation](https://stackoverflow.com/a/43938191)?

Reply:
That seems like a much cleaner solution. I've changed to that now. Thank you.

Reply:
Good idea on `skipif`. And I think for now it's good that these tests reach out to the cloud bucket, because we want to be sure that experience continues to work.